### DISCUSSION TITLE
nx has any talk groups like discord telegrapm?

### DISCUSSION THREAD

**Discussion by 0xZoomEye** [nrwl/nx#12330] [General]
RT

**Comment by twittwer** [comment]
https://nx.dev/community

There is a Slack Workspace: https://go.nrwl.io/join-slack

**Comment by ryancraigmartin** [comment]
The NX team just started a Discord Server [today](https://www.youtube.com/watch?v=yBsXB8xuTog). 
Join us here!
https://discord.gg/2Qj44Qf2BK
